internal/pkg/config/fleet.go

```diff
@@ -67,12 +67,12 @@ func strToLevel(s string) (zerolog.Level, error) {
 		l = zerolog.DebugLevel
 	case "info":
 		l = zerolog.InfoLevel
-	case "warning":
+	case "warn":
```

Comment:
Sorry, we didn't update the original issue but @cmacknz and I discussed that both `warn` and `warning` should be allowed